lib/datadog/tracing/tracer.rb

```diff
@@ -378,7 +374,7 @@ def start_span(
         # NOTE: This might be redundant sometimes (given #start_trace does this)
         #       however, it is necessary because the Context/TraceOperation may
         #       have been provided by a source outside the tracer e.g. OpenTracing
-        bind_trace_events!(trace)
+        # bind_trace_events!(trace) # Pending OpenTracing PR
```

Comment:
Can we uncomment this now? I think we need this for good operation still.

Reply:
Maybe you're right... probably don't need this now. Would probably be a small performance improvement too. Okay... remove this.